LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerHttpClient.cpp

```diff
@@ -252,6 +252,16 @@ void ULootLockerServerHttpClient::LogFailedRequestInformation(const FLootLockerS
 	if (!Data.IsEmpty()) {
 		LogString += FString::Format(TEXT("\n   Request Data: {0}"), { Data });
 	}
+#if WITH_EDITOR
+	if(!ResponseHeaders.IsEmpty())
+	{
+		LogString += FString::Format(TEXT("\n   -- Response Headers --"), { Data });
+		for (FString ResponseHeader : ResponseHeaders)
+		{
+			LogString += FString::Format(TEXT("\n     {0}"), { ResponseHeader });
+		}
+	}
+#endif
```

Comment:
Should the default be to always log all headers? Thinking that most of the time the user doesn't care about them.
We're just interested in the 🦝 (for now), so maybe just that?
But also, since this is server you might want all information all the time perhaps?

Reply:
Yeah, it's only for
- the server
- while in the editor
- and having set lootlocker logging to veryverbose

So I feel it's safe

Reply:
Do you disagree @JohannesLoot ?

Reply:
Nope, fully agree, a feature even I would say since it's only when set to veryverbose 👍